README.md

```diff
@@ -29,18 +29,27 @@ The displayed data is real and provided by [Quandl](https://www.quandl.com). The
 
 - Download or clone this repository locally
 - Ensure [Node.js](https://nodejs.org/), which includes npm, is installed
-- Ensure [Grunt](http://gruntjs.com/getting-started#installing-the-cli) is installed:
+
+
+Install dependencies:
 
 ```
-npm install -g grunt-cli
```

Comment:
As long as we keep grunt as a dev dependencies, the build should work just fine without having to ask the user to install it as a global module.

This is because npm adds `node_modules/.bin` to the path when running scripts:

https://docs.npmjs.com/misc/scripts#path

This is a great reason for adding npm scripts to run your build, tests, dev-servers etc ...